Core/Loaders/SimpleTileLoader.cs

```diff
@@ -0,0 +1,264 @@
+﻿using Microsoft.Xna.Framework;
```

Comment:
This entire file is a duplicate of something in a new namespace post-refactor, needs to be removed and refferences changed over to the new namespace